.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release on PyPI
+
+on:
+  push:
+    tags:
+    - v*
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: __token__
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
Could we please use the name PYPI_TOKEN for this instead?  This is more consistent with various other projects already using gh-actions.